src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationHandler.cs

```diff
@@ -24,7 +25,15 @@ public LinkedInAuthenticationHandler([NotNull] HttpClient client)
 
         protected override async Task<AuthenticationTicket> CreateTicketAsync([NotNull] ClaimsIdentity identity,
             [NotNull] AuthenticationProperties properties, [NotNull] OAuthTokenResponse tokens) {
-            var request = new HttpRequestMessage(HttpMethod.Get, Options.UserInformationEndpoint);
+            if (Options.Fields.Count == 0) {
```

Comment:
Just dug through their docs more. Looks like it will return just basic profile info if nothing is specified. So we can removed that check. I will go ahead and do that.

https://developer.linkedin.com/docs/signin-with-linkedin